sympy/physics/control/tests/test_lti.py

```diff
@@ -505,6 +506,23 @@ def test_Series_construction():
     raises(TypeError, lambda: Series(tf3, Matrix([1, 2, 3, 4])))
 
 
+def test_PIDController_creation_and_properties():
```

Comment:
Can you test it more extensively? 
I generally like if the code goes well with MATLAB unit tests / examples - https://in.mathworks.com/help/control/ref/pid.html

Reply:
Add some unit tests from the above link and add them to your test suite.